pkgs/top-level/octave-packages.nix

```diff
@@ -33,11 +33,6 @@ makeScope newScope (self:
       inherit computeRequiredOctavePackages;
     };
 
-    wrapOctave = callPackage ../development/interpreters/octave/wrap-octave.nix {
-      inherit octave;
-      inherit (pkgs) makeSetupHook makeWrapper;
-    };
```

Comment:
I think you need some way to track public API, which must not be removed.

Reply:
This is not exported anywhere, either it is only used occasionally, or it should be inherited in the attrSet below, or it should be removed.